content/030_self_guided_setup/32_request_credit.md

```diff
@@ -8,7 +8,7 @@ weight: 12
 <tr class="credit">
   <td class="credit" style="width:70%">To help you cover the AWS service costs incurred by doing this tutorial,
```

Comment:
Can we rollback changes on this file?  Doesn't seem necessary

Reply:
removed the lines. should we consider removing the file itself?